frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/StrategyDraggableItem/StrategyItem/StrategyExecution/StrategyExecution.tsx

```diff
@@ -32,7 +32,10 @@ export const StrategyExecution: FC<StrategyExecutionProps> = ({
     const { segments } = useSegments();
     const { isCustomStrategy, customStrategyParameters: customStrategyItems } =
         useCustomStrategyParameters(strategy, strategies);
-    const strategyParameters = useStrategyParameters(strategy, displayGroupId);
+    const strategyParameters = useStrategyParameters(
+        strategy as FeatureStrategySchema,
```

Comment:
This is where casting is needed, because `IFeatureStrategyPayload` allows for values other then `string` in 
```
interface IFeatureStrategyParameters {
    [key: string]: string | number | undefined;
}
```
where OpenAPI has
```
interface ParametersSchema {
    [key: string]: string;
}
```

Because `IFeatureStrategyParameters` is used in many other palces, it's not practical to change it now.